### ISSUE TITLE
Incorporating quality scores for ASE counts

### ISSUE THREAD

**Issue by rmagoglia** [TheFraserLab/ASEr#18]
Curious what others think about this - when we find a read overlapping a SNP and we can match it to its parent of origin, should we be checking the quality score of that base in the read (and have it pass a certain quality threshold) before counting that read towards the total counts for that parental allele? To me this seems more in keeping with other methods (e.g. mpileup) and might yield more reliable results.

**Comment by petercombs** [comment]
I think that's a reasonable idea, and it wouldn't even be that hard to implement.  I am skeptical that it would necessarily change a whole lot, since my understanding is that the vast majority of Illumina bases are of very high quality . Unless you have a some evidence pointing towards a lot of reads getting mis-categorized, I don't think it's worth your time.

**Comment by tehranchi** [comment]
Yep, that's what I do. It won't change "a lot" percentage wise but it will likely have an effect on the false positives regarding ASE calls.
